src/OpenTelemetry.OpAmp.Client/Internal/Listeners/IOpAmpListener.cs

```diff
@@ -3,12 +3,23 @@
 
 namespace OpenTelemetry.OpAmp.Client.Internal.Listeners;
 
-internal interface IOpAmpListener
+/// <summary>
+/// Marker interface for OpAmp listeners.
+/// </summary>
+public interface IOpAmpListener
 {
 }
 
-internal interface IOpAmpListener<TMessage> : IOpAmpListener
+/// <summary>
+/// A listener capable of handling OpAmp messages of a specific type.
```

Comment:
For non-code identifiers should we use `OpAMP` rather than `OpAmp`? Asking as it's written differently in the CHANGELOG.

Reply:
I believe all of these are fixed now.